src/main/java/com/baxance/songr/SongrController.java

```diff
@@ -34,43 +39,22 @@
                         "https://upload.wikimedia" +
                                 ".org/wikipedia/en/c/cf/Clipping._-_Splendor_%26_Misery.jpg")
         );
-        System.out.println(library.get(0).title);
-        System.out.println(library.get(1).artist);
-        System.out.println(library.get(2).length);
-
     }
 
+
+    @Autowired

```

Comment:
You should probably create a separate controller for albums instead of shoving everything in the same controller